### DISCUSSION TITLE
Airtable Import With Attachments [Feb 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5112] [Discord Threads]
> 🙋‍♂️ **Ayush Shinde**  
#### I have attached a screenshot too. Also not able to download those attachments. I have checked the logs too and I don't see any error there.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1061888526850986104/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Ayush Shinde**  
#### <@857998597474484295> While if I perform the same thing on localhost I am able to get all attachments easily.  
#  
> 🙋‍♂️ **Ayush Shinde**  
#### I have observed some errors in the console.  
![image0](https://cdn.discordapp.com/attachments/1061888527203303434/1061904530847051786/image.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### <@666284961619247124> Help us by creating a GitHub issue with available information (logs, screenshots). https://github.com/nocodb/nocodb/issues  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### Can you provide more details about your hosted setup? If I understand it right - import to localhost works alright.  
#  
> 🙋‍♂️ **Ayush Shinde**  
#### I have deployed it on caprover  
#  
> 🙋‍♂️ **Ayush Shinde**  
####   
![image0](https://cdn.discordapp.com/attachments/1061888527203303434/1061907825112137779/image.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1061888527203303434)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
